students/minjeong/users/views.py

```diff
@@ -50,17 +51,21 @@ def post(self, request):
 class LoginView(View):
     def post(self, request):
         try:
-            data     = json.loads(request.body)
+            data = json.loads(request.body)
     
             email    = data["email"]
-            password = data["password"]
+            password = data["password"].encode('utf-8')
 
-            user = User.objects.get(email = email)
+            user        = User.objects.get(email = email)
+            db_password = user.password.encode('utf-8')
 
-            if user.password != password:
-                return JsonResponse({"messages" : "INVALID_PASSWORD"}, status=400)
+            if not bcrypt.checkpw(password, db_password):
+                return JsonResponse({"messages" : "NOT_MATCH_PASSWORD"}, status=400)
 
-            return JsonResponse({"message" : "SUCCESS"}, status=200)
+            user_id           = user.id
+            access_token      = jwt.encode({'id' : user_id}, settings.SECRET_KEY, algorithm = settings.ALGORITHM)
```

Comment:
리뷰 적용이 안되었습니다. 확인해주세요!
```suggestion
            access_token      = jwt.encode({'id' : user.id}, settings.SECRET_KEY, algorithm = settings.ALGORITHM)
```

Reply:
아 settings 관련 내용 인줄 알았습니다^^ 수정했습니다.